Mathlib/Algebra/Group/Commute/Defs.lean

```diff
@@ -254,40 +253,3 @@ theorem mul_inv_cancel_assoc (h : Commute a b) : a * (b * a⁻¹) = b := by
 end Group
 
 end Commute
-
```

Comment:
I agree that these statements should go from a `Defs` file to a `Defs` file. I suppose you moved these because they are needed earlier?

Reply:
Two reasons actually:
1. They are needed earlier.
2. They are not about `Commute`. Their proofs use `Commute`, but can easily be made not to use it.